src/io/flutter/ProjectOpenActivity.java

```diff
@@ -59,6 +59,9 @@ public void runActivity(@NotNull Project project) {
     ApplicationManager.getApplication().executeOnPooledThread(() -> {
       sdk.queryFlutterConfig("android-studio-dir", false);
     });
+    if (FlutterUtils.isAndroidStudio() && !FLUTTER_PROJECT_TYPE.equals(ProjectTypeService.getProjectType(project))) {
```

Comment:
We already have similar code to this at the end of the method.

Is this code different? Or, should we unify the two cases somehow?

Reply:
Those lines got deleted by the co-edit PR. I should probably remove them here too.